### ISSUE TITLE
Error that appears to be related to SpeciesCat and SpeciesGroup

### ISSUE THREAD

**Issue by traceybit** [DanOvando/GUM#9]
Hey there!

I'm trying to help some researchers who are trying to run the GUM package for Panulirus argus stocks for which they have 35 years of catch data. We used the ISSCAAP code in the ASFIS 2016 Excel document to determine the species category and group (43 and "Lobsters, spiny-rock lobsters"). When we run the GUM package, we get the following error:

Error in Data$SpeciesCatName[Where] <- ClosestGroup$SpeciesCatName : 
  replacement has length zero
In addition: Warning messages:
1: In min(GroupDistance) : no non-missing arguments to min; returning Inf
2: In max(ClosestCategories$SpeciesCat) :
  no non-missing arguments to max; returning -Inf

Any suggestions for troubleshooting?

Thanks,
Tracey

**Comment by tclavelle** [comment]
Are you manually entering the ISSCAAP and scientific name before running the code? What are you putting for the common name? If you're currently leaving the common name blank, try putting something in.

**Comment by traceybit** [comment]
Yes, and we are using the English common name for the common name -- here is an excerpt from the code used to prep the data:

lobster_df <- lobster %>%
  mutate(SpeciesCat = 43,    # Fill in Species Category. Follow steps in PDF document.
  SpeciesGroup = "Lobsters, spiny-rock lobsters",  # Fill in Species Group. Follow steps in PDF document.
  CommName = "Caribbean spiny lobster")

And here is what the data input file looks like:
![image](https://cloud.githubusercontent.com/assets/11774384/23878856/09b3fb40-0807-11e7-997d-7f7ebdfd98cb.png)

**Comment by tclavelle** [comment]
Check to make sure there's no blank entries, leading/trailing spaces, and that `is.numeric(lobster_df$SpeciesCat)==TRUE` and `is.character(lobster_df$SpeciesCatName) == TRUE`

**Comment by traceybit** [comment]
Yup checked for that but still getting the same error. Is there another reason why the package might not run? I can send you the csv if that would help.

**Comment by DanOvando** [comment]
Yeah send the csv please, I'll take a look  a little later this week
